CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/time/LayerByTimePlugin.java

```diff
@@ -441,7 +437,6 @@ public void read(final GraphReadMethods rg, final PluginInteraction interaction,
                 //Tag on the back of this loop and update the visibility of each node to match it's z value
                 if (isTransactionLayers) {
                     float vxZ = wgcopy.getFloatValue(zAttr, vertexId);
-                    wgcopy.setFloatValue(vxVisibilityAttr, vertexId, vxZ / (z + 1));
                 }
```

Comment:
@Polaris5514 from my reading of the code I think this entire if block could be removed. What do you think?